### DISCUSSION TITLE
Contributing some features

### DISCUSSION THREAD

**Discussion by somespecialone** [Gobot1234/steam.py#85] [Ideas]
Hi! I want to create small project, and need some features, that i can made and do a pull request, but want to ask firstly. I think `trade.Item` class needs `owner` attribute/property that must be a `client.User` instance of owner. Why? Trade offers contains received item, inspected item from CSGO can be item not from ours backpack. Also, on `trade.Item` model i need `actions` attr which contains actions from data dict - to extract inspect url in csgo-ext. I will do this anyway in my forks, but want to improve/get better your library, so what do you think?

**Comment by Gobot1234** [comment]
I'm not entirely convinced by the first point as any item in items_to_recieve is never going to be our own, but adding actions to item sounds like a good idea to me.

**Reply by somespecialone** [reply]
I mean that with `owner` attribute/async property we can interact with owner `client.User` instance. As i understand, now we can't directly see who owns item (that need to check trade offer and his owner if item in `items_to_recieve`), but what about `BackpackItem` returned from `inspect` method of csgo-ext `Client`? I want to implement this `inspect` method, actually, and think `owner` must be good choice.

**Reply by Gobot1234** [reply]
owner is already a thing on csgo.BackpackItem and is the 32 bit ID of the owner

**Comment by Gobot1234** [comment]
Actually yeah an owner attribute sounds like a great idea, I'll implement this myself.
